internal/service/flexcluster/resource_test.go

```diff
@@ -72,6 +78,7 @@ func checksFlexCluster(projectID, clusterName string, terminationProtectionEnabl
 		"termination_protection_enabled": fmt.Sprintf("%v", terminationProtectionEnabled),
 	}
 	checks = acc.AddAttrChecks(resourceName, checks, attributes)
+	checks = acc.AddAttrChecks(dataSourceName, checks, attributes)
```

Comment:
probably it's a good idea to add some more checks for the computed attributes, such as backup_settings, cluster_type etc. I think that only checking that it has a value would be enough in this case, but we can check specific values also